src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Config/ConditionalConfig.cs

```diff
@@ -32,130 +42,40 @@ public IEnumerable<IOperationProvider> ConfigureFromJObject(JObject rawConfigura
             {
                 operations = ConditionalBlockCommentConfig.ConfigureFromJObject(rawConfiguration);
             }
-            else
+            else if (string.Equals(commentStyle, "builtin", StringComparison.OrdinalIgnoreCase))
```

Comment:
Add a check to catch loops in definitions - a built-in could theoretically reference another built-in, or itself - which is an authoring error.